app/PickMe_nebula0/app/build.gradle.kts

```diff
@@ -45,44 +45,46 @@ dependencies {
     implementation("com.google.firebase:firebase-messaging")
     implementation("com.google.firebase:firebase-storage:20.2.1")
 
-    // Image loading
-    implementation("com.squareup.picasso:picasso:2.8")
-
-    // QR code support with ZXing
-    implementation("com.journeyapps:zxing-android-embedded:4.1.0") {
-        exclude("com.google.protobuf", "protobuf-lite")
-    }
-
-    // UI and Compose dependencies
+    // AndroidX dependencies
     implementation("androidx.activity:activity-compose:1.9.3")
-    implementation("com.google.android.material:material:1.4.0")
-    implementation("androidx.viewpager2:viewpager2:1.0.0")
```

Comment:
are these removed because theyre unused?

Reply:
no It should be samewhere in gradle file